pyuvsim/uvsim.py

```diff
@@ -276,101 +272,39 @@ def initialize_uvdata(uvtask_list, source_list_name, uvdata_file=None,
         history += (' Based on config files: ' + obs_param_file + ', '
                     + telescope_config_file + ', ' + antenna_location_file)
 
-    history += ' Npus = ' + str(mpi.get_Npus()) + '.'
-
-    task_freqs = []
-    task_bls = []
-    task_times = []
-    task_antnames = []
-    task_antnums = []
-    task_antpos = []
-    task_uvw = []
-    ant_1_array = []
-    ant_2_array = []
-    telescope_name = uvtask_list[0].telescope.name
-    telescope_location = uvtask_list[0].telescope.location.geocentric
-
-    source_0 = uvtask_list[0].source
-    freq_0 = uvtask_list[0].freq
-    for task in uvtask_list:
-        if not task.source == source_0:
-            continue
-        task_freqs.append(task.freq)
-
-        if task.freq == freq_0:
-            task_bls.append(task.baseline)
-            task_times.append(task.time)
-            task_antnames.append(task.baseline.antenna1.name)
-            task_antnames.append(task.baseline.antenna2.name)
-            ant_1_array.append(task.baseline.antenna1.number)
-            ant_2_array.append(task.baseline.antenna2.number)
-            task_antnums.append(task.baseline.antenna1.number)
-            task_antnums.append(task.baseline.antenna2.number)
-            task_antpos.append(task.baseline.antenna1.pos_enu)
-            task_antpos.append(task.baseline.antenna2.pos_enu)
-            task_uvw.append(task.baseline.uvw)
-
-    antnames, ant_indices = np.unique(task_antnames, return_index=True)
-    task_antnums = np.array(task_antnums)
-    task_antpos = np.array(task_antpos)
-    antnums = task_antnums[ant_indices]
-    antpos = task_antpos[ant_indices]
-
-    freqs = np.unique(task_freqs)
-
-    uv_obj = UVData()
+    history += ' Npus = ' + str(mpi.Npus) + '.'
 
-    # add pyuvdata version info
-    history += uv_obj.pyuvdata_version_str
```

Comment:
Is the input uv guaranteed to already have both the pyuvdata and pyuvsim version strings?